app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :title do "Forgot Password" end %>
```

Comment:
This template has some content tweaks to make it work well with our application/ devise layout.